lib/spack/spack/platforms/cray_xc.py

```diff
@@ -50,9 +51,9 @@ def setup_platform_environment(self, pkg, env):
             similar to linux/standard linker behavior
         """
         env.set('CRAYPE_LINK_TYPE', 'dynamic')
-        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
-        if os.path.isdir(cray_wrapper_names):
-            env.prepend_path('PATH', cray_wrapper_names)
+#        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
+#        if os.path.isdir(cray_wrapper_names):
+#            env.prepend_path('PATH', cray_wrapper_names)
```

Comment:
This is ongoing work on a more permanent fix, I’m just trying to quickly get something in that will work for now.

From: "Adam J. Stewart" <notifications@github.com<mailto:notifications@github.com>>
Reply-To: LLNL/spack <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 21, 2016 at 2:15 PM
To: LLNL/spack <spack@noreply.github.com<mailto:spack@noreply.github.com>>
Cc: Gregory Becker <becker33@llnl.gov<mailto:becker33@llnl.gov>>, Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: Re: [LLNL/spack] Crayxc detection (#1335)


In lib/spack/spack/platforms/cray_xc.py<https://github.com/LLNL/spack/pull/1335#discussion_r71788544>:

> @@ -50,9 +51,9 @@ def setup_platform_environment(self, pkg, env):
>              similar to linux/standard linker behavior
>          """
>          env.set('CRAYPE_LINK_TYPE', 'dynamic')
> -        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
> -        if os.path.isdir(cray_wrapper_names):
> -            env.prepend_path('PATH', cray_wrapper_names)
> +#        cray_wrapper_names = join_path(spack.build_env_path, 'cray')
> +#        if os.path.isdir(cray_wrapper_names):
> +#            env.prepend_path('PATH', cray_wrapper_names)


I would delete this instead of commenting it out. You don't need to worry about losing the old code, that's the whole point of version control.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/LLNL/spack/pull/1335/files/f0d5317913d0c47806e102126ea67aacbcf4db80#r71788544>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ANUwcGOkYsOxbDDCJVBBfnO533mGE42Wks5qX-GMgaJpZM4JSOKt>.